packages/jaeger-ui/src/components/TraceDiff/TraceDiff.tsx

```diff
@@ -187,7 +187,9 @@ export function mapDispatchToProps(dispatch: Dispatch<any>) {
   return { fetchMultipleTraces, forceState };
 }
 
-export default connect<TStateProps, TDispatchProps, TOwnProps, ReduxState>(
-  mapStateToProps,
-  mapDispatchToProps
-)(TraceDiffImpl);
+export default withRouteProps(
```

Comment:
could you please add documentation to this function (should've asked in the prev PR)?

https://github.com/jaegertracing/jaeger-ui/blob/bcee38a2157d6ac1fe573e463d17239810684773/packages/jaeger-ui/src/utils/withRouteProps.tsx#L28

Reply:
Documentation in the form of jsDocs comments added throughout the function? Or is there any dedicated documentation guide for each function used in Jaeger-UI?